recipes/perl-try-tiny/LICENSE

```diff
@@ -0,0 +1,32 @@
+This software is Copyright (c) 2009 by יובל קוג'מן (Yuval Kogman).
```

Comment:
This file can be removed, the `license: LICENSE` in `meta.yaml` should pick up and package the file that's in the source tarball

Reply:
@cbrueffer This seems to be a problem. Removing `LICENSE` file requires me to remove the `license_file` key from `meta.yaml`, otherwise the build fails. If I remove that key, the linter fails (see below). What do you suggest?